src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,22 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
```

Comment:
While this works, the logic isnt that clear. Since there are only 3 changes (for 0, 1, and 2 or more years old) you should only need 3 if statements:
```
  if (number === 0) {
    return [number, number, number];
  }
  if (number === 1) {
    return [number, number + 14, number + 14];
  }
  if (number > 1) {
    return [number, number * 4 + 16, number * 5 + 14];
  }
```